### PR TITLE
ZX-169 - Ability to upload photo for contacts

### PR DESCRIPTION
**Pull request by celinakhalife**
GraphQL changes for https://github.com/ZimbraOS/zm-x-web/pull/437

### CODE REVIEW

**Comment by pl12133** [comment]
@celinakhalife I see you need write access to this repo, so I talked to @jebady16 + @dajay0 and we decided to move this repo to the ZimbraOS org where you will have write access. That should be happening in the coming days.

For now, please continue using your fork for contributing. Thank you!

**Comment by celinakhalife** [comment]
@silentsakky if there are no issues on this PR, can we merge it?

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -207,23 +208,34 @@ export class ZimbraBatchClient {
...
-			body: {
-				account: {
-					by: 'name',
-					_content: username
-				},
-				oldPassword: password,
-				password: loginNewPassword
-			}
-		});
+		public changePassword = ({
```
Comment (pl12133) [inline src/batch-client/index.ts]: Indentation is off by one tab.

**Comment by michaelmedellin** [comment]
@pl12133 How can we move forward with closing this PR?

**Comment by keithmancuso** [comment]
hey @pl12133 celina updated the errors so this "should" be ok to merge

**Pull request merged**